README.md

```diff
@@ -38,5 +34,7 @@ After the build you will have a self-contained binary (`geras`). It writes logs
 ## Limitations
 
 * PromQL supports queries without `__name__`. This is not allowed in geras and it will raise an error.
-* Geras loads the metric names from OpenTSDB and keeps them in memory for queries like `__name__=~""`.
-* Have to use `:` instead of `.` in metric names (PromQL limitation).
+* Geras periodically loads metric names from OpenTSDB and keeps them in memory to support queries like `{__name__=~""}`.
+* Thanos' primary timeseries backend is Prometheus, which doesn't support `.` in metric names. However OpenTSDB metrics generally use `.` as a seperator within names. In order to query names containing a `.` you will need to either:
+  * Replace all `.` with `:` in your query
+  * Use the magic `__name__` label to specify the metric name, e.g. `{__name__="cpu.percent"}`
```

Comment:
Line #39 and #40 are solutions for the same limitation. IMO it would be better connect them with an `or`.